CTA_Instance.py

```diff
@@ -42,19 +46,30 @@ def run_post_analyzer(self):
         populate self.final_warnings based on analysis of final version
         '''
         post_analyser = PostAnalyser(self.cta_version_root)
-        self.initial_warnings = post_analyser.get_all_warnings()
+        self.final_warnings = post_analyser.get_all_warnings()
+        self.cta_recommendations = post_analyser.get_all_recommendation()
 
     def save_to_csvs(self):
         '''
         Store initial_warnings, final_warnings and changes as separate csvs,
         so that they can be inspected after the run.
         :return: path to newly generated folder where these csvs are stored (next to presentation folder)
         '''
-        pass
+        root = self.report_root + "/raw_data"
+        folder = os.path.exists(root)
+        if not folder:
+            os.mkdir(root)
+        pd.DataFrame(self.initial_warnings).to_csv(root + '/pre-analyzer.csv')
+        pd.DataFrame(self.final_warnings).to_csv(root + '/post_analyzer.csv')
```

Comment:
maybe "initial_warnings.csv"/ "final_warnings.csv" is a more helpful term for the user, who does not know anything about pre/post analyzer logic.